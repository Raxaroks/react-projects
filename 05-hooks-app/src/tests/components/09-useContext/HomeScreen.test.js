import '@testing-library/jest-dom';
import { mount } from 'enzyme';
import HomeScreen from '../../../components/09-useContext/HomeScreen';
import { UserContext } from '../../../components/09-useContext/UserContext';


describe('Pruebas en el componente: HomeScreen', () => {

    const user = {
        name: 'Andr√©s',
        mail: 'andres@enchingue.com'
    }

    const componentWrapper = mount(
        <UserContext.Provider value={{ user }}>
            <HomeScreen/>
        </UserContext.Provider>
    );

    test('Debe de renderizarse correctamente.', () => {
        expect(componentWrapper).toMatchSnapshot();
    });    

});
