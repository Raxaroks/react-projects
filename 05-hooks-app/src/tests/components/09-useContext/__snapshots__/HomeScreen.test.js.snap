// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente: HomeScreen Debe de renderizarse correctamente. 1`] = `
<div>
  <h1>
    HomeScreen: Página de inicio
  </h1>
   
  <hr />
  <pre>
    {
   "name": "Andrés",
   "mail": "andres@enchingue.com"
}
  </pre>
</div>
`;
